README.md

```diff
@@ -57,7 +57,7 @@ Executing this Spark job will output the following:
 You don't need to be Scala developer to use ADAM. You could also run the following ADAM CLI command for the same result:
 
 ```bash
-$ adam-submit count_kmers \
+$ adam-submit -- count_kmers \
```

Comment:
echoing @laserson's other comment, it would be nice if we didn't require the ` -- ` here and elsewhere for the most basic kinds of ADAM invocations.